apps/package.json

```diff
@@ -14,6 +14,7 @@
     "test:unit": "grunt unitTest",
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
+    "test:interpreter": "./node_modules/@code-dot-org/js-interpreter-tyrant/bin/run.js --run --diff --root ./test/interpreter --interpreter ./test/interpreter/patched-interpreter.js --progress",
```

Comment:
Ugh, what a headache.  Leave it for now.  Hopefully we'll remember to clean it up after we upgrade.